test/helpers.js

```diff
@@ -417,6 +417,17 @@ export async function setInputValues(page, fields) {
     return await new Promise((r) => setTimeout(r, 300));
 }
 
+export async function setSelectDropdownValue(page, selector, optionValue) {
+    return await page.evaluate(
+        ({ selector, ST_ROOT_SELECTOR, optionValue }) => {
+            const select = document.querySelector(ST_ROOT_SELECTOR).shadowRoot.querySelector(selector);
+            select.value = optionValue;
+            select.dispatchEvent(new Event("change", { bubbles: true }));
```

Comment:
Is there no way of doing this with `page.select`?

Reply:
Puppeteer's page.select() method doesn't natively support selection within shadow roots, 
hence using this approch

Reply:
not 100 sure, but please check if something like this works:
```
    const ele = await waitForSTElement(page, "[data-supertokens~=selectX]");
    ele.select(...)
 ```